script.transmission/resources/lib/search.py

```diff
@@ -0,0 +1,129 @@
+import sys
+import re
+import socket
+import requests
+from urllib.parse import quote, quote_plus, urlencode
+from bs4 import BeautifulSoup
+
+socket.setdefaulttimeout(15)
+
+class Search:
+    def __init__(self):
+        return NotImplemented
+    def search(terms):
+        return NotImplemented
+
+class TPB(Search):
```

Comment:
Oh well, not allowed at all!

Reply:
Drop any piracy related functionality from the addon if you want it on our repo.